foundation/org.eclipse.persistence.core/.classpath

```diff
@@ -5,7 +5,8 @@
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8"/>
 	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/cdi.api_1.0.0.v201105160744.jar"/>
 	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/javax.jms_1.1.0.v200906010428.jar"/>
-	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/org.glassfish.javax.json_1.0.4.v201311181159.jar" sourcepath="/ECLIPSELINK_HOME/plugins/org.glassfish.javax.json.source_1.0.4.v201311181159.jar"/>
+	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/javax.json-api_1.1.2.jar"/>
+	<classpathentry kind="var" path="ECLIPSELINK_HOME/plugins/javax.json_1.1.2.jar"/>
```

Comment:
core does not (and should not) depend on any particular implementation. BTW: this kind of things is better done in an extra commit